### DISCUSSION TITLE
PLEASE READ - DO NOT UPGRADE PMS! Latest Version of Plex not Scraping

### DISCUSSION THREAD

**Discussion by JPH71** [CodyBerenson/PGMA-Modernized#232] [General]
If you have upgraded your Plex Media Server Software and it stops scraping giving you an error similar to
**unknown error (_ssl.c:2938)**

You can download an older version of PMS from this website - 
https://www.videohelp.com/software/Plex/old-versions

I went back to PlexMediaServer-1.31.3.6868-28fc46b27-x86_64.exe, but it was the very latest one that did me in.

**Comment by fourstix** [comment]
The latest update for QNap is PlexMediaServer-1.32.0.6918-6f393eda1-x86_64.qpkg.  Is that the one that caused the problems?

**Comment by JPH71** [comment]
I sent the picture to the github page... its the one after that caused the
issue that I downloaded from the Windows app store...

**Comment by fourstix** [comment]
Thanks. I'll guess I'll pass on upgrading PMS until the dust settles.   Is there an issue open for this in the Plex forums?

**Comment by JPH71** [comment]
Not at the moment.. I just gave advice, try using the latest pms and see if
it affects you in the nearl future and then downgrade your version ....

We will need to test this on various OS... etc...

**Comment by ChgoChad** [comment]
> If you have upgraded your Plex Media Server Software and it stops scraping giving you an error similar to **unknown error (_ssl.c:2938)**
> 
> You can download an older version of PMS from this website - https://www.videohelp.com/software/Plex/old-versions
> 
> I went back to PlexMediaServer-1.31.3.6868-28fc46b27-x86_64.exe, but it was the very latest one that did me in.

I can confirm the latest version **(ver 1.32.0.6918)** definitely breaks things on **Windows**.  None of the **PGMA-Modernized** agents will scrape successfully, each one throwing the error mentioned above. (**_ssl.c:2938**). Many of my plugins stopped working as well. The release notes for that version do not call out many significant changes to this version, but whatever they changed breaks a lot of stuff. 

Downgrading to previous version (**ver 1.31.3.6868**) immediately solved the problem.

**Comment by JPH71** [comment]
I tried to upgrade again yesterday but the newest version still breaks the
plugins...
Thanks for confirming this xxxx

**Comment by jfreiman** [comment]
I just upgraded from ver 1.31.3.6868 to the latest beta for Windows Version 1.32.0.6950 and, just like before, no matches are being found/made with IAFD, it is however being matched by GEVI -- which doesn't have or match the model artwork.

[com.plexapp.agents.IAFD.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11256237/com.plexapp.agents.IAFD.log)
[com.plexapp.agents.GEVI.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11256238/com.plexapp.agents.GEVI.log)

**Reply by JPH71** [reply]
Now that is weird - I tried again 3 days ago - till not matching on any - so reverted to the old version as before.

**Comment by jfreiman** [comment]
Is this closed because PGMA now works with the current 1.32.xx builds or because the new versions are no longer supported by PGMA?
